### DISCUSSION TITLE
Support for vulnerabilityAlerts on Azure DevOps

### DISCUSSION THREAD

**Discussion by Tasin5541** [renovatebot/renovate#33252] [Suggest an Idea]
Azure DevOps can generate vulnerability reports through [GitHub Advanced Security](https://azure.microsoft.com/en-us/products/devops/github-advanced-security). Is there any chance that support for `vulnerabilityAlerts` can be extended to Azure DevOps by utilizing the vulenarability report from `GitHub Advanced Security`?

Endpoint for fetching Azure DevOps alerts: https://learn.microsoft.com/en-us/rest/api/azure/devops/advancedsecurity/alerts/list?view=azure-devops-rest-7.2

**Comment by rarkins** [comment]
Probably yes, if someone can confirm that necessary APIs are available

**Reply by Tasin5541** [reply]
Does this help? 
https://learn.microsoft.com/en-us/rest/api/azure/devops/advancedsecurity/instances/list?view=azure-devops-rest-7.2

**Reply by rarkins** [reply]
That looks like "get an alert" and not "list all alerts". You can test using curl or a tool like postman to verify

**Reply by Tasin5541** [reply]
My bad, I provided the wrong link.
This one fetches all the alerts: https://learn.microsoft.com/en-us/rest/api/azure/devops/advancedsecurity/alerts/list?view=azure-devops-rest-7.2

**Reply by rarkins** [reply]
Please update the original discussion body so that it can be converted into an issue with all information included

**Reply by Tasin5541** [reply]
Done, thanks.

**Comment by rarkins** [comment]
Converted to issue https://github.com/renovatebot/renovate/issues/33477
